test/views/behaviours/StringViewBehaviours.scala

```diff
@@ -63,7 +63,7 @@ trait StringViewBehaviours extends QuestionViewBehaviours[String] {
         "show an error summary" in {
 
           val doc = asDocument(createView(form.withError(error)))
-          assertRenderedById(doc, "error-summary-title")
+          assertRenderedByClass(doc, "error-summary-title")
```

Comment:
Change "error-summary-title" to "govuk-error-summary"

Reply:
Done